assets/javascripts/discourse/widgets/discourse-reactions-counter.js.es6

```diff
@@ -89,6 +89,19 @@ export default createWidget("discourse-reactions-counter", {
 
   click(event) {
     if (!this.capabilities.touch || !this.site.mobileView) {
+      event.stopPropagation();
+
+      // in case we lost sync due to another widget not in the same tree
+      // collapsing the panel, we attempt to reconciliate from DOM state
+      const container = document.getElementById(this.buildId(this.attrs));
```

Comment:
Ideally another widget would change the state and not add the class. Is this fixable?

Reply:
Yes I think it is, it's the meaning of my commit message. I will attempt to refactor this deeper.